src/test/scala/uk/gov/homeoffice/drt/utils/DateUtilSpecs.scala

```diff
@@ -0,0 +1,48 @@
+package uk.gov.homeoffice.drt.utils
+
+import org.scalatest.flatspec.AsyncFlatSpec
+import org.scalatest.matchers.must.Matchers
+
+class DateUtilSpecs extends AsyncFlatSpec with Matchers {
+
+  import DateUtil._
+
+
+  "Compare date check" should "be true for same date and different time" in {
+    val date1 = DateUtil.parseDate("2018-11-23 21:35:00")
+    val date2 = DateUtil.parseDate("2018-11-23 22:35:00")
+
+    isSameDate(date1, date2) mustEqual true
+  }
+
+  "Compare date check" should "be true for same date and one no time" in {
```

Comment:
'be true where the dates are the same and one has a time and the other has no time'